pkg/availability/multisigcollector/multisigcollector.go

```diff
@@ -66,10 +65,12 @@ func NewReconfigurableModule(mc *ModuleConfig, nodeID t.NodeID, logger logging.L
 			func(mscID t.ModuleID, params *factorymodulepb.GeneratorParams) (modules.PassiveModule, error) {
 
 				// Extract the IDs of the nodes in the membership associated with this instance
-				m := params.Type.(*factorymodulepb.GeneratorParams_MultisigCollector).MultisigCollector.Membership
+				mscParams := params.Type.(*factorymodulepb.GeneratorParams_MultisigCollector).MultisigCollector
+				m := mscParams.Membership
```

Comment:
`m` is only used in the next line. Maybe we can get rid of that variable and use `mscParams.Membership` directly?

Reply:
done